src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -12062,10 +12172,11 @@ static class E
     [Fact]
     public void InstanceMethodInvocation_Simple_ExpressionTree()
     {
-        // Tracked by https://github.com/dotnet/roslyn/issues/76130 : decide whether to allow expression tree scenarios. Verify shape of the tree if we decide to allow
         var source = """
```

Comment:
It looks like we are not verifying the shape of the tree as the prototype comment suggested.